app/assets/javascripts/availability.js.coffee

```diff
@@ -0,0 +1,29 @@
+jQuery ->
+  window.availability_updater = new AvailabilityUpdater
+class AvailabilityUpdater
+  constructor: ->
+    this.request_availability()
+  availability_url: "http://bibdata.princeton.edu/availability"
+  request_availability: ->
+    ids = this.record_ids().toArray()
+    params = $.param({ids: ids})
+    url = "#{@availability_url}?#{params}"
+    $.getJSON(url, this.process_records)
+  process_records: (records) =>
+    for record_id, availability_info of records
+      this.apply_record(record_id, availability_info)
+  apply_record: (record_id, availability_info) ->
+    return if availability_info.more_holdings == true
```

Comment:
Eventually we might want to provide more of an explanation for why the user needs to click on the record (ie. more copies/items in a given holding location vs more holding locations), but I think "Click on record for full availability" is sufficient for this PR.